packages/blade/src/components/Table/types.ts

```diff
@@ -82,7 +82,13 @@ type TableProps<Item> = {
    * The onSelectionChange prop is a function that is called when the selection changes.
    * The function is called with an object that has a values property that is an array of the selected rows.
    **/
-  onSelectionChange?: ({ values }: { values: TableNode<Item>[] }) => void;
+  onSelectionChange?: ({
+    values,
+    selectedIds,
```

Comment:
Why do we need both?

Reply:
It'll be a breaking change if we remove the values